### ISSUE TITLE
FastifyDeprecation: You are decorating Request/Reply with a reference type. 

### ISSUE THREAD

**Issue by borodadada** [fastify/help#333]
Hello. After update to "fastify": "^3.9.1" i got this message.
```
(node:14820) [FSTDEP006] FastifyDeprecation: You are decorating Request/Reply with a reference type. 
This reference is shared amongst all requests. 
Use onRequest hook instead. Property: cookies
```
How fix it?

**Comment by L2jLiga** [comment]
Hi there, here's PR which deprecates type references: https://github.com/fastify/fastify/pull/2688 and here's an issue https://github.com/fastify/fastify/issues/2598, solution - do not use type references in request/reply decorators

**Comment by mcollina** [comment]
I've just released fastify-cookie 4.1.1 that fixes it.

**Comment by borodadada** [comment]
now work fine, thanks
